### PR TITLE
preparing for 1.4.0 release stuff

### PR DESCRIPTION
**Pull request by acharb**
some cleanup for the 1.4.0 release

tested building / using now and looks good with these changes

### CODE REVIEW

**Comment by acharb** [comment]
there was an extra yarn.lock in the `km` project that I deleted, it's not needed

File: @stellar/typescript-wallet-sdk-km/webpack.config.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ module.exports = (env = { NODE: false }) => ({
       stream: require.resolve("stream-browserify"),
       url: require.resolve("url"),
       util: require.resolve("util"),
+      vm: require.resolve("vm-browserify"),
```
Comment (acharb) [inline @stellar/typescript-wallet-sdk-km/webpack.config.js:25]: webpack 5 is throwing annoying warning saying we need to polyfill this now, so added

File: @stellar/typescript-wallet-sdk-km/package.json [hunk 2]
```diff
@@ -47,6 +47,8 @@
     "prepare": "husky install",
     "test": "jest --watchAll",
     "test:ci": "jest --ci",
-    "build": "webpack --config webpack.config.js"
+    "build:web": "webpack --config webpack.config.js",
```
Comment (acharb) [inline @stellar/typescript-wallet-sdk-km/package.json:50]: km should build for web and node like the regular sdk is

File: @stellar/typescript-wallet-sdk/webpack.config.js [hunk 3]
```diff
@@ -32,4 +35,9 @@ module.exports = (env = { NODE: false }) => ({
     path: path.resolve(__dirname, "lib"),
   },
   target: env.NODE ? "node" : "web",
+  plugins: [
```
Comment (acharb) [inline @stellar/typescript-wallet-sdk/webpack.config.js:38]: also needed to add this for React environment build

(not sure why wasn't needed before, maybe bc webpack version upgraded for this PR)

**Review by Ifropc — APPROVED** [review]

**Pull request merged**